src/test/java/dev/snowdrop/release/services/ReleaseFactoryTest.java

```diff
@@ -209,28 +209,41 @@ private void validate(Release release) {
         var description = component.getDescription();
         assertTrue(description.contains(component.getParent().getVersion()));
         assertFalse(description.contains("**")); // this would happen if some substitutions didn't happen
+        assertTrue(description.contains("It is important that you also communicate the EOL of your product so that we can plan our future releases accordingly."));
+        assertFalse(description.contains("tested and supported"));
+        assertTrue(description.contains("- product name:"));
         for (Artifact artifact : artifacts) {
             assertTrue(description.contains(artifact.getName()));
             assertTrue(description.contains(artifact.getVersion()));
         }
 
-        component = components.get(7);
+        component = components.get(2);
+        assertEquals("AMQP", component.getName());
+        assertTrue(component.getDescription().contains("It is important that you also communicate the EOL of your product so that we can plan our future releases accordingly."));
+        assertTrue(component.getDescription().contains("tested and supported"),component.toString() + component.getDescription());
+        assertFalse(component.getDescription().contains("- product name:"),component.toString() + component.getDescription());
+        component = components.get(6);
         assertEquals("Narayana starter", component.getName());
+        assertTrue(component.getDescription().contains("tested and supported"));
+        assertFalse(component.getDescription().contains("It is important that you also communicate the EOL of your product so that we can plan our future releases accordingly."));
+        assertFalse(component.getDescription().contains("- product name:"));
+        component = components.get(7);
+        assertEquals("Vert.X starter", component.getName());
         final var product = component.getProduct();
         assertNotNull(product);
-        assertEquals("Narayana", product.getName());
-        assertEquals("Currently supported Narayana version information", product.getTitle());
+        assertEquals("Vert.X", product.getName());
```

Comment:
should be `Vert.x` and `Vert.X` ;-)

Reply:
I'll create an issue to follow this as this requires changes to the `release.yml` files located at the BOM repository.

Reply:
done.